pkg/tool/aws-cli/aws-cli.go

```diff
@@ -141,6 +154,10 @@ func (t *Tool) symlinkPath(latestDir string) string {
 	return filepath.Join(latestDir, "aws")
 }
 
+func (t *Tool) symlinkCompleterPath(latestDir string) string {
```

Comment:
**thought;** there's no reason for this to be a method vs a normal function, and it probably wouldn't need to be split in a new function because it's a one liner. However, it matches the current codebase, so I'd say it's fine :)